submissions/antonkottans/js-dom/script.js

```diff
@@ -0,0 +1,136 @@
+document.body.onload = () => {
+    initializeContentArray = () => {
```

Comment:
Use `const` to store variables like this function.

Reply:
Are you going to store your function in window object? In such simple case? 
Please, move it into your file closure.

Reply:
i didnt know i can do that thing... fixed